src/components/widgets/One.vue

```diff
@@ -1,3 +1,143 @@
 <template>
-	<div></div>
-</template>
\ No newline at end of file
+  <div class="widget-one">
+      <div v-bind="info" class="widget-one__date">{{info.date}}</div>
+    <div>
+      <h2 class="widget-one__header">NASA</h2>
+    </div>
+    <div v-bind="info" class="widget-one__title">{{info.title}}</div>
+    <div class="widget-one__button-container">
+      <button v-on:click="toggleFunction" class="widget-one__button">{{buttonText}}</button>
+    </div>
+    <div class="widget-one__content-container">
+      <div v-if="toggle === true">
+        <img v-bind:src="info.url" class="widget-one__image">
+      </div>
+      <div v-else>
+        <p v-bind="info" class="widget-one__explanation">{{info.explanation}}</p>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+
+console.log(import.meta.env)
+export default {
+  name: 'WidgetOne',
+  data () {
+    return {
+      info: {},
+      toggle: true,
+      buttonText: 'Explanation'
+    }
+  },
+  mounted: function() {
```

Comment:
Read a few articles about doing api calls in `created` rather than `mounted`. These two articles offers good explanations:

https://jerickson.net/created-vs-mounted-in-vue/
https://www.digitalocean.com/community/tutorials/vuejs-component-lifecycle

WDYT?

Reply:
changed to `created`